src/ResourceManager/AzureStack/Admin/AzureStack.Management.Commerce.Admin/Module/AzureStack.Management.Commerce.Admin.csproj

```diff
@@ -0,0 +1,67 @@
+﻿<?xml version="1.0" encoding="utf-8"?>
```

Comment:
I think these belong in src\StackAdmin  since they are not AzureRm modules.  Also, why doesn't this build into the STack directory?

Reply:
Those where supposed to be deleted.  Will delete them.